.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--
+  Thanks for contributing!
+
+  Provide a description of your changes below and a general summary in the title
```

Comment:
I believe adding the phrase - "of your PR"  after "title" will clear some doubts, given that it might be a little confusing to know exactly what title is referred to here